web/src/user_group_edit_members.ts

```diff
@@ -191,17 +228,17 @@ function add_new_members({pill_user_ids}) {
         user_id_set.delete(current_user.user_id);
     }
 
-    let ignored_deactivated_users;
-    let ignored_already_added_users;
+    let ignored_deactivated_users: User[] | undefined;
+    let ignored_already_added_users: User[] | undefined;
```

Comment:
Would it be cleaner to just initialize these as empty lists, rather than allowing `undefined`? Worth testing that, can be a follow-up little PR.